app/src/main/java/com/berniesanders/fieldthebern/screens/Main.java

```diff
@@ -62,6 +63,16 @@ public String getScopeName() {
     return Main.class.getName();
   }
 
+  public static final Creator<Main> CREATOR = new ScreenCreator<Main>() {
+    @Override protected Main doCreateFromParcel(Parcel source) {
```

Comment:
Hey Patrick, thanks for looking this over. 

I'm not sure what to do in these cases where a field isn't in the existing class constructor, but instead is getting populated somewhere in the Presenter.  Should I add a constructor that uses the field(s) so that the Creator can populate it?  Or does the fact they were left out earlier imply they're not the parceler's concern?